ge0/parser.hpp

```diff
@@ -6,6 +6,9 @@
 
 namespace ge0
 {
+extern std::array<std::string_view, 6> const kGe0Prefixes;
+extern std::array<std::string_view, 3> const kLegacyMwmPrefixes;
```

Comment:
Can avoid exposing these arrays by introducing a function. See the comment above.

Reply:
Reworked.